### DISCUSSION TITLE
How to check if an `op` is skipped or successful?

### DISCUSSION THREAD

**Discussion by ivanov-slk** [dagster-io/dagster#5677] [Q&A]
Previously, a `solid` had the properties `success` and `skipped` that allowed checking for these conditions, e.g. in tests:
```
result = execute_pipeline(pipeline=my_pipeline,
                                         run_config=my_config)

self.assertTrue(result.success)
self.assertTrue(result.result_for_solid('solid_1').success)
self.assertTrue(result.result_for_solid('solid_2').skipped)
```
I think that this could be replicated by searching through `ExecuteInProcessResult.all_node_events`, however this requires some filtering and checking. Is there a more concise way of making the above checks?

**Comment by sryza** [comment]
@dpeng817
